ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/config/LinearRaoParameters.java

```diff
@@ -74,6 +84,15 @@ public LinearRaoParameters setMaxIterations(int maxIterations) {
         return this;
     }
 
+    public ObjectiveFunction getObjectiveFunction() {
+        return objectiveFunction;
+    }
+
+    public LinearRaoParameters setObjectiveFunction(ObjectiveFunction objectiveFunction) {
```

Comment:
Do you really need the setter to return something? (and not just void?)

Reply:
This pattern is used for all the configs of PowSyBl.

We were also surprised about that, but we discussed about it with @TheMaskedTurtle to PowSyBl experts which said it was useful in some cases to chain the setters... After this discussion we decided to keep the PowSyBl logics, even if it is not useful for us it gives more "uniformity". :p